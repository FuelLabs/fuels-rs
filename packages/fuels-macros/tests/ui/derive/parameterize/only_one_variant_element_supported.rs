use fuels_macros::Parameterize;

#[derive(Parameterize)]
enum SomeEnum {
    // problem because no elements present
    B(),
}

#[derive(Parameterize)]
enum AnotherEnum {
    A(u64, u32),
}

fn main() {}
