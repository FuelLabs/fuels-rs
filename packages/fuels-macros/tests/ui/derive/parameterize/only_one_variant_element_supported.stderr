error: must have exactly one element
 --> tests/ui/derive/parameterize/only_one_variant_element_supported.rs:6:6
  |
6 |     B(),
  |      ^^

error: must have exactly one element
  --> tests/ui/derive/parameterize/only_one_variant_element_supported.rs:11:6
   |
11 |     A(u64, u32),
   |      ^^^^^^^^^^
